internal/netsync/manager.go

```diff
@@ -1762,18 +1790,15 @@ func (m *SyncManager) OnNotFound(peer *Peer, notFound *wire.MsgNotFound) {
 		// from the global requested maps.
```

Comment:
No need to distinguish global/per-peer maps any longer.

Reply:
Good call.  Updated.